README.md

```diff
@@ -44,7 +44,7 @@ instructions for adding the correct package database for your OS version, and
 then:
 
 ```bash
-$ apt-get install llvm-6.0-dev
+$ apt-get install llvm-7.0-dev
```

Comment:
The package has been renamed, it’s now only called `llvm-7-dev`.

Reply:
Oops! I'm not on a linux machine so I missed this. Updated the patch